slides/05_First_Components.html

```diff
@@ -432,35 +428,137 @@ <h3>Solution continued</h3>
         <section>
           <section>
             <h2>List</h2>
+          </section>
+          <section data-auto-animate>
             <p>
-              To render a list, we apply the
+              To render a list of items, we call the
               <span class="code">map</span>
-              function on a list to produce HTML-Elements
-            </p>
-            <p>
-              Each list item shall have a unique
-              <a
-                href="https://reactjs.org/docs/lists-and-keys.html#keys"
-                target="_blank"
-              >
-                key
-              </a>
+              function on an array.<br />In the callback we map the array item
+              to a JSX element.
             </p>
+
+            <p>We can map to an HTML element or to a custom component.</p>
             <!-- prettier-ignore -->
-            <pre><code class="hljs javascript" data-trim data-line-numbers="6|1-12"><script type="text/template">
-                        function List() {
+            <pre data-id="code"><code class="hljs javascript" data-trim data-line-numbers="5-7|"><script type="text/template">
+                        export default function List() {
                             const items = ['Apple', 'Orange', 'Mango'];
```

Comment:
consider setting this const items outside of the components, so the items aren't recreated on every render.